package.json

```diff
@@ -107,6 +108,6 @@
     "whatwg-fetch": "^3.5.0"
   },
   "engines": {
-    "node": ">= 10.0.0"
+    "node": ">= 14.0.0"
```

Comment:
Would this constitute a major breaking change?

Reply:
It's likely, 🤔  
cc @ljharb

Reply:
Yes, absolutely and always. why is this needed?